python_modules/libraries/dagster-dbt/dagster_dbt/asset_decorator.py

```diff
@@ -206,6 +206,53 @@ def my_dbt_assets(context: OpExecutionContext, dbt: DbtCliResource, config: MyDb
                     dbt_build_args += ["--full-refresh"]
 
                 yield from dbt.cli(dbt_build_args, context=context).stream()
+
+        Creating partitions for dbt assets:
```

Comment:
To get the partition def docs linked:
```suggestion
        Defining Dagster :py:class:`~dagster.PartitionDefinition` alongside dbt:
```